proto/dydxprotocol/clob/query.proto

```diff
@@ -192,6 +197,39 @@ message StreamUpdate {
   uint32 exec_mode = 5;
 }
 
+// SubaccountPerpetualPosition provides information on a subaccount's updated
+// perpetual positions.
+message SubaccountPerpetualPosition {
+  // The `Id` of the `Perpetual`.
+  uint32 perpetual_id = 1;
+  // The size of the position in base quantums.
+  uint64 quantums = 2;
+}
+
+// SubaccountAssetPosition provides information on a subaccount's updated asset
+// positions.
+message SubaccountAssetPosition {
+  // The `Id` of the `Asset`.
+  uint32 asset_id = 1;
+  // The absolute size of the position in base quantums.
+  uint64 quantums = 2;
+}
+
+// StreamSubaccountUpdate provides information on a subaccount update. Used in
+// the full node GRPC stream.
+message StreamSubaccountUpdate {
```

Comment:
Maybe add a comment about if the snapshot will be for every perpetual/asset or only the subacc positions that have been updated

Reply:
done